client/src/pages/Home.js

```diff
@@ -1,11 +1,26 @@
 import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 import MainTable from "../Component/MainTable";
+import SMSModal from "../Component/SMSModal";
 
 import "./Home.css";
 
+
+
 export function Home() {
 	const [message, setMessage] = useState("Loading...");
+	const [checkedCheckboxes, setChecked] = useState([]);
+
+	const onHandleSelectedUserState = (isSelected, userID ) => {
```

Comment:
```suggestion
	const onHandleSelectedUserState = (isSelected, clientId) => {
```
We're using the name `client_id` in the code so for consistency and lack-of-surprise we should keep it the same here